src/renderer/di.ts

```diff
@@ -16,6 +16,9 @@ import { diRendererSymbolTable as diSymbolTable } from "readium-desktop/renderer
 import { RootState } from "readium-desktop/renderer/redux/states";
 import { initStore } from "readium-desktop/renderer/redux/store/memory";
 import { Store } from "redux";
+
+import * as mainApp from "./components/App";
+import * as readerApp from "./components/reader/App";
```

Comment:
I would like to suggest changing this to a cleaner syntax:
`import MainApp from "./components/App";`

...and simply use `MainApp` instead of `mainApp.default` in the code below.

Reply:
1e07d8e9